sycl/doc/GettingStartedXilinxFPGA.md

```diff
@@ -853,7 +853,9 @@ cd $SYCL_HOME
 # You can also try --branch sycl/unified/next for a bleeding edge experience
 git clone --branch sycl/unified/master git@github.com:triSYCL/sycl.git llvm
 python $SYCL_HOME/llvm/buildbot/configure.py
-python $SYCL_HOME/llvm/buildbot/compile.py
+python $SYCL_HOME/llvm/buildbot/compile.py --xrt
+# The --xrt is to enable the optional XRT plugin. it is a replacement for the OpenCL plugin.
+# Because the OpencCL implementation in XRT is not maintained anymore
```

Comment:
```suggestion
# Use --xrt is to enable the optional XRT plugin. This is a replacement for the OpenCL plugin
# because XRT offers more control and expressiveness on the hardware
python $SYCL_HOME/llvm/buildbot/compile.py --xrt
```
to be more factual